runtime/src/anchor/mod.rs

```diff
@@ -43,24 +43,18 @@ const CHILD_INFO: ChildInfo<'static> = ChildInfo::new_default(b"anchor");
 #[derive(Encode, Decode, Default, Clone, PartialEq)]
 #[cfg_attr(feature = "std", derive(Debug))]
 pub struct PreCommitData<Hash, AccountId, BlockNumber> {
-    signing_root: Hash,
-    identity: AccountId,
-    expiration_block: BlockNumber,
+    pub signing_root: Hash,
+    pub identity: AccountId,
+    pub expiration_block: BlockNumber,
```

Comment:
why do we need to make this public? Are we using these anywhere ?

Reply:
Just make the definition consistent and ready for access in the future.